.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Don't modify line endings when comitting / pulling
+*.* binary
```

Comment:
Ah man, its 2020, does this problem not have a simple and easy solution yet!?

Anyway, one option is to do nothing and hope people commit the right things, but I think it would be better to encourage / enforce lf line endings to be committed, as problems can occur otherwise.

I have read https://git-scm.com/docs/gitattributes, and it seems like `* text eol=lf` will do the right thing. We could also list out the file types

Do you agree?

Reply:
Yeah that sounds like a good plan! I've seen `* text eol=lf` used quite a bit. And it _is_ indeed annoying that this is still an issue.